ipywidgets/static/widgets/js/widget.js

```diff
@@ -628,8 +636,29 @@ define(["nbextensions/widgets/widgets/js/utils",
                 this.update_attr('border-radius', this._default_px(this.model.get('border_radius')));
 
                 this.update_css(this.model, this.model.get("_css"));
+                
+                this.setStyle(this.model.get('style'));
             }, this));
         },
+        
+        serializers: _.extend({
+            style: {deserialize: unpack_models},
+        }, WidgetModel.serializers),
+        
+        setStyle: function(style, oldStyle) {
```

Comment:
I probably should try to do this, but it gets tricky.  I'll try doing this and ping back.